packages/core/src/App/Containers/Layout/header/dtrader-header.tsx

```diff
@@ -86,32 +82,32 @@ const DTraderHeader = observer(() => {
         >
             <div className='header__menu-items'>
                 <div className='header__menu-left'>
-                    {!is_mobile && (
+                    {isDesktop && (
```

Comment:
isDesktop is same as !isMobile? I see you using both across the files, maybe if it's same, avoid using negations: just use isDesktop and isMobile.

Reply:
I use both interchangeably depends on the use case. Like someplaces i need to show it for tablet and mobile, so i use !isDesktop. In some places i need to show it for desktop and tablet so i use !isMobile

Reply:
Alright, in that case it makes sense to use negations